docs/docs/architecture/adr/adr-003-ica-controller.md

```diff
@@ -0,0 +1,35 @@
+# ADR 003: Interchain Accounts Controller Module
+
+## Changelog
+
+- 2024-03-08: Initial Draft
+
+## Status
+
+DRAFT
+
+## Abstract
+
+The Interchain Accounts Controller IBC module allows users of one chain to create and control accounts on other chains. The Hub currently doesn't have ICA Controller module enabled, so it is not possible to create accounts on other chains from the Hub chain.
+
+## Context
+
+Enabling the ICA Controller module on the Hub would support various use cases. One such case could be the provider-based governance that would allow the ATOM stakers to participate in a governance on consumer chains.
+
+## Decision
+
+The ICA Controller module will be included in the application, so the Hub will have both ICA Host and Controller modules. The implementation will use the Controller module's built-in authentication mechanism, since we don't have a need for custom authentication logic. According to this, users will directly use `MsgRegisterInterchainAccount` and `MsgSendTx` messages defined by the Controller module. The possibility provided by the Controller module to define underlaying application to have custom processing of IBC messages exchanged by the Controller module (e.g. `OnChanOpenInit`, `OnAcknowledgementPacket`, etc.) will not be used, since there is currently no need for this.
+
+## Consequences
+
+### Positive
+
+Users of the Hub will have a possibility to create and utilize Interchain Accounts on other IBC connected chains.
+
+### Negative
+
+### Neutral

```

Comment:
Maybe put a few sentences about not having to create custom auth modules to implement ICA (as was previously the case).

Reply:
Added in https://github.com/cosmos/gaia/pull/2994/commits/c5f88348a7b4266afec580c0c871a4b02e8de9df